src/test/java/org/alliancegenome/curation_api/HTPExpressionDatasetAnnotationFmsITCase.java

```diff
@@ -0,0 +1,95 @@
+	}
+
+	private final String htpDatasetBulkPostEndpoint = "/api/htpexpressiondatasetannotation/bulk/FB/htpexpressiondatasetannotationfile";
+	private final String htpDatasetTestFilePath = "src/test/resources/bulk/fms/09_htp_dataset/";
+	private final String htpDatasetFindEndpoint = "/api/htpexpressiondatasetannotation/find?limit=100&page=0";
+
+	@Test
+	@Order(1)
+	public void htpDatasetBulkUploadCheckFields() throws Exception {
+		checkSuccessfulBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "AF_01_all_fields.json", 2);
+
+		RestAssured.given().when().header("Content-Type", "application/json").body("{}").post(htpDatasetFindEndpoint)
+				.then().log().all().statusCode(200).body("totalResults", is(1)).body("results", hasSize(1))
+				.body("results[0].htpExpressionDataset.curie", is("FB:FBlc0003342"))
+				.body("results[0].htpExpressionDataset.crossReferences", hasSize(2))
+				.body("results[0].htpExpressionDataset.crossReferences[0].referencedCurie", is("TEST:TestMol00000001"))
+				.body("results[0].htpExpressionDataset.preferredCrossReference.referencedCurie", is("TEST:TestMol00000001"))
+				.body("results[0].htpExpressionDataset.secondaryIdentifiers", hasSize(1))
+				.body("results[0].htpExpressionDataset.secondaryIdentifiers[0]", is("GEO:GSE38764"))
+				.body("results[0].references", hasSize(2))
+				.body("results[0].references[0].crossReferences[0].referencedCurie", is("PMID:25920554"))
+				.body("results[0].subSeries", hasSize(1))
+				.body("results[0].subSeries[0].curie", is("FB:FBlc0003342"))
+				.body("results[0].subSeries[0].crossReferences", hasSize(2))
+				.body("results[0].numberOfChannels", is(2))
+				.body("results[0].name", is("TEST TITLE"))
+				.body("results[0].relatedNote.freeText", is("TEST_SUMMARY"))
+				.body("results[0].categoryTags[0].name", is("anatomical structure"));
+	}
+
+	@Test
+	@Order(2)
+	public void htpDatasetBulkUploadMissingRequiredFields() throws Exception {
+
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "MR_01_no_dataset_id.json",2,1,0);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "MR_02_no_title.json",2,1,1);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "MR_03_no_category_tags.json",2,1,1);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "MR_04_no_primary_id.json",2,1,0);
+	}
+
+	@Test
+	@Order(3)
+	public void htpDatasetBulkUploadEmptyRequiredFields() throws Exception {
+
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "ER_01_empty_dataset_id.json",2,1,0);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "ER_02_empty_title.json",2,1,1);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "ER_03_empty_category_tags.json",2,1,1);
+		checkFailedBulkLoad(htpDatasetBulkPostEndpoint, htpDatasetTestFilePath + "ER_04_empty_primary_id.json",2,1,0);
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChA1MzIxYTk0M2QyZTEyYTAzEgpjaGVja3N0eWxl -->